src/website/views/custom-elements/button.pug

```diff
@@ -0,0 +1,301 @@
+    <chi-button color='primary' type='pill' outline>Primary</chi-button>
+    <chi-button color='secondary' type='pill' outline>Secondary</chi-button>
+    <chi-button color='danger' type='pill' outline>Danger</chi-button>
+    <chi-button color='dark' type='pill' outline>Dark</chi-button>
+
+h3 Icon buttons
+p.-text
+  | For interfaces with limited space, an <code>icon</code> button can
+  | be used for common actions such as editing, deleting, closing, etc.
+.example.-mb--3
+  .-p--3
+    chi-button(type='icon')
+      .a-icon
+        chi-icon(icon='atom')
+    chi-button(flat, extra-class='-ml--2', type='icon')
+      .a-icon
+        chi-icon(icon='atom')
+  :code(lang='html')
+    <chi-button type='icon' >
+      <div class="a-icon">
+        <chi-icon icon='atom' />
+      </div>
+    </chi-button>
+
+    <!-- flat -->
+    <chi-button flat type='icon' >
+      <div class="a-icon">
+        <chi-icon icon='atom' />
+      </div>
+    </chi-button>
+
+h4 Labeled icon buttons
+.example.-mb--3
+  .-p--3
+    chi-button()
+      .a-icon
+        chi-icon(icon='atom')
+      span Button
+    chi-button(extra-class='-mr--2 -ml--2')
+      span Button
+      .a-icon
+        chi-icon(icon='atom')
+    chi-button()
+      .a-icon
+        chi-icon(icon='atom')
+      span Button
+      .a-icon
+        chi-icon(icon='atom')
+  :code(lang='html')
+    <!-- left aligned -->
+    <chi-button>
+      <div class="a-icon">
+        <chi-icon icon='atom' />
+      </div>
+      <span>Button</span>
+    </chi-button>
+
+    <!-- right aligned -->
+    <chi-button>
+      <span>Button</span>
+      <div class="a-icon">
+        <chi-icon icon='atom' />
+      </div>
+    </chi-button>
+
+    <!-- both sides -->
+    <chi-button>
+      <div class="a-icon">
+        <chi-icon icon='atom' />
+      </div>
+      <span>Button</span>
+      <div class="a-icon">
+        <chi-icon icon='atom' />
+      </div>
+    </chi-button>
+
+h4 Floating icon buttons
+.example.-mb--3
+  .-p--3.-d--flex
+    each type in ['', 'primary', 'secondary', 'danger', 'dark']
+      div.-pr--2
+        chi-button(color=type, type='float')
+          .a-icon
+            chi-icon(icon='atom')
+  :code(lang='html')
+    <chi-button type='float'>
+      <div class="a-icon">
+        <chi-icon icon='atom' />
+      </div>
+    </chi-button>
+
+    <chi-button type='float' color='primary'>
+      <div class="a-icon">
+        <chi-icon icon='atom' />
+      </div>
+    </chi-button>
+
+    <chi-button type='float' color='secondary'>
+    <div class="a-icon">
+    <chi-icon icon='atom' />
+    </div>
+    </chi-button>
+
+    <chi-button type='float' color='danger'>
+      <div class="a-icon">
+        <chi-icon icon='atom' />
+      </div>
+    </chi-button>
+
+    <chi-button type='float' color='dark'>
+      <div class="a-icon">
+        <chi-icon icon='atom' />
+      </div>
+    </chi-button>
+
+
+h4 Close icon button
+
+.example.-mb--3
+  .-p--3
+    chi-button(type='close')
```

Comment:
![Untitled](https://user-images.githubusercontent.com/43439749/57613708-dc5ab180-7577-11e9-97ac-4a6f9dbc6290.png)

How to solve? Custom elements should be configured to have the same display value as the components they wrap. 
File: src/custom-elements/src/components/button/button.scss
:host(chi-button) should have display: inline-flex;
File: src/custom-elements/src/components/icon/icon.scss
:host(chi-icon) should have display: inline-flex; //This has already been solved in other PR that are not merged yet, but please, change it also here to be able to check everything is ok.

Reply:
> ![Untitled](https://user-images.githubusercontent.com/43439749/57613708-dc5ab180-7577-11e9-97ac-4a6f9dbc6290.png)
> 
> How to solve? Custom elements should be configured to have the same display value as the components they wrap.
> File: src/custom-elements/src/components/button/button.scss
> :host(chi-button) should have display: inline-flex;
> File: src/custom-elements/src/components/icon/icon.scss
> :host(chi-icon) should have display: inline-flex; //This has already been solved in other PR that are not merged yet, but please, change it also here to be able to check everything is ok.

@srinath-koneru Review this comment, please.